### PR TITLE
shansen3030 updating .md file caption

### LINKED ISSUE #3: Add A New Slide
## We Need More Slides
Grab another octocat and create a new slide from scratch.
If you need help, check out the Appendix in the training manual.

### PR DESCRIPTION
**Pull request by shansen3030**
## PR - shansen3030 -10/13/2022 - Git Training

[ ] Check this box to get $100.00
[ ] Check this box to get $1000.00

This resolves #3

### CODE REVIEW

**Comment by shansen3030** [comment]
Not sure if i need to add "-" in front of checkbox ..

File: _slides/13-shansen3030.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Scott is starting to "git" this!
```
Comment (andreakward) [inline _slides/13-shansen3030.md:6]: Awesome!  I would like $1,000 dollars haha

**Review by andreakward — APPROVED** [review]
Approved

File: _slides/13-shansen3030.md (hunk 1 above)
Comment (shansen3030) [reply _slides/13-shansen3030.md:6]: Thanks for the Approval.   $1000.00 check is in the mail.  👍

**Comment by shansen3030** [comment]
PR for shansen3030 has successfully merged and confirmed.

**Pull request merged**